provider/cmd/pulumi-resource-aws/schema.json

```diff
@@ -135584,28 +135584,28 @@
                     "items": {
                         "type": "string"
                     },
-                    "description": "List of headers allowed.\n"
+                    "description": "Specifies which headers are allowed.\n"
```

Comment:
Where do those changes come from? I don't see those descriptions being modified in the PR

Reply:
> Fix an issue with documentation override: due to a bug docs/resource/aws_s3_bucket_legacy.md was not being picked up and the aws.s3.Bucket page was incorrectly serving docs for the aws.s3.BucketV2 resource. This is fixed here.

Reply:
There was a bug that sent BucketV2 documentation into the Bucket page, ignoring the handcrafted aws_s3_bucket_legacy.md file

Reply:
Ah, ok that makes sense!